### ISSUE TITLE
Change language for datetime picker

### ISSUE THREAD

**Issue by AmusedMe** [xgfe/react-native-datepicker#91]
Thank you for your time.
I look through all the issues, just one about language. I still not find how to specific one language.

**Comment by feyy** [comment]
I just use the react native Component, I think the component's language is same as your mobile system.

**Comment by Clowning** [comment]
Yes it's strange, when the modal is open, the language is support but when the value is printed in the component, it's in English!

**Comment by ozsirma** [comment]
https://github.com/facebook/react-native/pull/15635/files

You can examine this pull request.

**Comment by assafb81** [comment]
@ozsirma,
is it merge to main?

**Comment by ozsirma** [comment]
It is usable with v0.52.0 that merged the pull request. @assafb81

**Comment by assafb81** [comment]
how RN v0.52.0 is relevant?
I thought it is react-native-datepicker issue

**Comment by ozsirma** [comment]
> react native datePicker component for both Android and IOS, using DatePickerAndroid, TimePickerAndroid and **DatePickerIOS**

DatePickerIOS is native component for iOS. Therefore if you use this feature, you should use v52.0 or you should change files manually in your node_modules. @assafb81

**Comment by assafb81** [comment]
@ozsirma thanks

**Comment by pablo-albaladejo** [comment]
My fork include this feature:
https://github.com/pablo-albaladejo/react-native-datepicker

<DatePicker
   locale={'es'}
   date={this.state.date}
   mode='date'
   format="DD-MM-YYYY">

**Comment by jenskuhrjorgensen** [comment]
Nice, @pablo-albaladejo :)

However, I need support for Android too. This should be supported by RN. Upvote this feature request to help push this feature through :D

**Comment by pablo-albaladejo** [comment]
Thanks @jenskuhrjorgensen,

There is no issue at the Android Datepicker locale; it just read the system locale. Screenshots of the component:

<img width="483" alt="locale_es" src="https://user-images.githubusercontent.com/7994467/37037989-d8c52416-2153-11e8-9f73-916e6b21c07a.png">
<img width="479" alt="locale_en" src="https://user-images.githubusercontent.com/7994467/37038008-e7235208-2153-11e8-8d5a-1334cacc79eb.png">

**Comment by jenskuhrjorgensen** [comment]
Yes @pablo-albaladejo, but I would like to set it manually - is that possible?

**Comment by fhadsheikh** [comment]
I'm also wondering how to manually set locale

**Comment by karthikeyan110798** [comment]
> Thanks @jenskuhrjorgensen,
> 
> There is no issue at the Android Datepicker locale; it just read the system locale. Screenshots of the component:
> 
> <img alt="locale_es" width="483" src="https://user-images.githubusercontent.com/7994467/37037989-d8c52416-2153-11e8-9f73-916e6b21c07a.png">
> 
> <img alt="locale_en" width="479" src="https://user-images.githubusercontent.com/7994467/37038008-e7235208-2153-11e8-8d5a-1334cacc79eb.png">

**Comment by karthikeyan110798** [comment]
Hello sir ,
Iam trying to change the language of date picker based on user selection , could you suggest  some examples.
Thanks a lot!

**Comment by AhmSaeed** [comment]
Can I set a customize date picker locale to follow my app language not system language on Android is there any available solution or work around?

**Comment by ZainaliSyed** [comment]
Has anyone got this solution for **ANDROID Language change Manually** 

even community Library has same issue for android
